### ISSUE TITLE
Some emails from IMailServer are not parsed since v4.22.2p3

### ISSUE THREAD

**Issue by azumakuniyuki** [sisimai/p5-Sisimai#323]
The following sample emails are not parsed since v4.22.2p3.
- email-imailserver-02.eml
- email-imailserver-03.eml
- email-imailserver-04.eml

This bug is in the Perl version of Sisimai only.

**Comment by azumakuniyuki** [comment]
Both of bounce mails from AmazonSES above are parsed with `delivered => 1` parameter and returns like the following:
```
            deliverystatus   "2.6.0",
            destination      "simulator.amazonses.com",
            diagnosticcode   "250 2.6.0 Message received",
```

**Comment by azumakuniyuki** [comment]
`rfc3464-28.eml` also be parsed with `delivered => 1` parameter and returns like the following result.

```
            deliverystatus   "2.1.5",
            destination      "neko.example.jp",
            diagnosticcode   "250 2.1.5 Ok",
```

**Comment by azumakuniyuki** [comment]
Fixed and released (v4.25.2)
https://blog.libsisimai.org/post/186698215086/v4252-bug-fix
